lexicons/bsky.app/getRepostedBy.json

```diff
@@ -15,6 +15,8 @@
       "required": ["uri", "repostedBy"],
       "properties": {
         "uri": {"type": "string"},
+        "cid": {"type": "string"},
```

Comment:
I added `cid` here to unify it with `getLikedBy`.   If that wasn't the intention just give a shout 👍